docs/publishers/core-datasets.md

```diff
@@ -0,0 +1,135 @@
+# Core Datasets
+
+**Core Datasets** - important, commonly-used datasets in high quality, easy-to-use & open form.
+
+**High Quality & Reliable** - sourcing, normalizing and quality checking a set of **key reference and indicator datasets such as country codes, currencies, GDP and population**
```

Comment:
This info is taken from http://data.okfn.org/roadmap/core-datasets. Either we can keep it or change it in their main page. Personally, I think, we can keep it as a 'normalizing'